### ISSUE TITLE
Build and publish the app as a snap/ flatpak

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/flutter#213]
(Low prio issue)

The app should build without much effort as a linux desktop application (we probably want to work a bit on #111 before). We could think about publishing on [snapcraft](https://snapcraft.io/build) and [flatpak](https://docs.flatpak.org/en/latest/) if all plugins work there.

**Comment by 12people** [comment]
I can help with the Flatpak. (Not a fan of Snap, due to its server side being proprietary.)

Here's a manifest for a Flutter Flatpak I released that might serve as inspiration: https://github.com/flathub/com.enjoyingfoss.Parlera

**Comment by rolandgeider** [comment]
yes, that would be great (but after we release the exercise stuff 😅 )

**Comment by Ombrelin** [comment]
I too would love a release of the app for the desktop via flatpak !

**Comment by rolandgeider** [comment]
Some TODOs

- [ ] adding desktop support doesn't currently work out of the box https://github.com/flutter/flutter/issues/53106
- [ ] uploading pictures to the gallery didn't seem to work

**Comment by rolandgeider** [comment]
pull request sent https://github.com/flathub/flathub/pull/4075
